libs/utils/utils.go

```diff
@@ -131,3 +132,21 @@ func InterfaceSliceToStringSlice(originalSlice []interface{}) []string {
 	}
 	return newSlice
 }
+
+// RemoveDuplicatesFromSlice ...
+func RemoveDuplicatesFromSlice(list []string) string {
+	cleaned := []string{}
+	m := make(map[string]bool)
+
+	for _, item := range list {
+		if item == "" {
+			continue
+		}
+		if _, ok := m[item]; !ok {
```

Comment:
What if the comparison fails for some string with leading/trailing space? For example, "**User1**" and "**User1** " (with a trailing space), are two different values. When we are comparing/storing the first string it would work fine but when it comes to the second string (assuming the first string is already present in the map), this statement would set `ok` equal to `false` and thereby giving incorrect results, we should trim the spaces from the item before searching it in the map.

Reply:
Good catch!
Fixed that.